core/chains/evm/gas/rollups/l1_oracle.go

```diff
@@ -49,10 +54,11 @@ func IsRollupWithL1Support(chainType chaintype.ChainType) bool {
 	return slices.Contains(supportedChainTypes, chainType)
 }
 
-func NewL1GasOracle(lggr logger.Logger, ethClient l1OracleClient, chainType chaintype.ChainType, daOracle evmconfig.DAOracle) (L1Oracle, error) {
+func NewL1GasOracle(lggr logger.Logger, ethClient l1OracleClient, chainType chaintype.ChainType, daOracle evmconfig.DAOracle, _ map[string]Client) (L1Oracle, error) {
```

Comment:
Shouldn't we be using the map here? I know the implementation to use the map comes later but we should at least have the check to make sure the `daOracle.L1ChainID` is present in the map. Think I saw you mention that check in another comment.